src/js/store.js

```diff
@@ -209,3 +209,4 @@ export const anyRecordingShortcut = writable(false);
 export const videoShortcutAction = writable('');
 export const currentlyEditingPreset = writable(-1);
 export const promptToShow = writable(/** @type {String[]} */ ([]));
+export const scrollTo = writable({ top: 0 });
```

Comment:
This should be typed with `ScrollToOptions`.

Reply:
Fixed in [5ccc1ca](https://github.com/LiveTL/LiveTL/pull/408/commits/5ccc1ca8919c134aa56ff174f67e3bcef3e167b4)